pyproject.toml

```diff
@@ -6,4 +40,45 @@ exclude = '''
 (
   onnxslim/argparser.py
 )
-'''
\ No newline at end of file
+'''
+
+[dependency-groups]
+dev = [
+    "coverage",
+    "onnxruntime<1.20.0 ; python_version<'3.10'",
+    "onnxruntime>=1.20.0 ; python_version>='3.10'",
+    "pytest",
+    "pytest-xdist",
+    "timm",
+    "torchvision",
+    # required by torch but apparently not declared?
+    "onnxscript",
```

Comment:
onnxscript only supports python >= 3.9, but we are committed to support python 3.8 because our downstream projects need it, like [ultralytics](https://github.com/ultralytics/ultralytics), maybe we can constrain torch version, I believe onnxscript in introduced very lately.

Reply:
Don't worry, I will fix it.